tests/wamr-test-suites/tsan_suppressions.txt

```diff
@@ -0,0 +1,11 @@
+# Proposing to accept this risk for now. As I understand, it might be wasi-libc related
+# https://github.com/bytecodealliance/wasm-micro-runtime/pull/1963#issuecomment-1455342931
+race:STORE_U32
+
+
+# https://github.com/bytecodealliance/wasm-micro-runtime/issues/2680
+race:execute_post_instantiate_functions
```

Comment:
I will try to fix the issue, we can remove this suspension later when it is fixed.

Reply:
yeah but now we can merge that and put a note to #2680 in order not to forget about it